modules/ossm-deploy-multi-mesh.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/ossm-deploy-mod-v2x.adoc
+
+
+[id="ossm-deploy-multi-mesh_{context}"]
+= Multimesh or federated deployment model
+
+[role="_abstract"]
```

Comment:
Unless something changed and I missed it, I don't think we're supposed to be adding in these Pv2 items yet. (Here and elsewhere in the PR)

Reply:
OK, cool, I'll pull them out then!